app/src/main/res/layout/activity_main.xml

```diff
@@ -7,9 +7,158 @@
     tools:context="com.example.mpvcalculatorrx.MainActivity">
 
     <TextView
-        android:layout_width="wrap_content"
+        android:id="@+id/text_display"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_centerInParent="true"
-        android:text="Hello World!"/>
+        android:layout_alignParentTop="true"
+        android:gravity="center_vertical|end"
+        android:textSize="@dimen/calc_display_text_size"/>
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@id/text_display"
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="50dp"
+            android:orientation="horizontal"
+            android:weightSum="1">
+
+            <Button
+                android:id="@+id/button_7"
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_weight=".25"
+                android:text="7"/>
```

Comment:
Extract the hard-coded strings to strings.xml

Reply:
Done